src/screens/FeedScreen/Content.js

```diff
@@ -15,6 +15,7 @@ import {fonts, normalizeFontSizeByWidth} from '../../utils/fonts';
 import {getCaptionWithTopicStyle} from '../../utils/string/StringUtils';
 import useCalculationContent from './hooks/useCalculationContent';
 import {getCommentLength} from '../../utils/getstream';
+import dimen from '../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `dimen` is added but it's not clear from this hunk where it's used. Please ensure that it's used in the code and not an unnecessary import.
```diff
+ import dimen from '../../utils/dimen';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->